src/app.js

```diff
@@ -156,15 +159,39 @@ var run = function () {
   chromeCloudSync()
 
   // ----------------- editor ----------------------
-
   var editor = new Editor()
-
-  // ----------------- tabbed menu -------------------
+  // -------------- fileExplorer ------------------
+  // @NOTE HTML#ID creates window['<ID name>'] element as a global
```

Comment:
every browser automatically creates `window.nameofid` as a global reference to the DOM element with the id `nameofid`, for example `<div id="nameofid">hello world</div>`

It's a comment to explain where the `window.treeview` in line 166 comes from - i think not every developer might now about this behavior.